src/app/modules/trip/components/self-trips/self-trips.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="header-text">
```

Comment:
this is practically the same template as the `list-trips`, is there any way to reuse it?

Reply:
Should we study angular, and apply to a job of that nature?

Reply:
XD